libvuln/opts.go

```diff
@@ -154,6 +161,29 @@ func (o *Opts) parse(ctx context.Context) error {
 	return nil
 }
 
+func remoteMatchersFunc(o *Opts) error {
+	for _, m := range o.RemoteMatchers {
+		if m["name"] == "crda" {
+			m, err := crda.NewMatcher(WithParams(m["params"]))
+			if err == nil {
+				o.Matchers = append(o.Matchers, m)
+			}
+		}
+	}
+	return nil
+}
+
+func WithParams(params string) string {
```

Comment:
WithParam should go to crda/remotematcher.go

Reply:
Okay.I can do that but according to the function prototype you had written earlier.This seemed like what you expected.

Reply:
So we will be passing the params to NewMatcher as argument right?

Reply:
Thats what I meant over here in https://github.com/arajkumar/claircore/pull/4#discussion_r485535129

Reply:
Refer pyupio/updaterset.go